docs/subnets/create-a-simple-rust-vm.md

```diff
@@ -0,0 +1,1256 @@
+# How to Build a Simple Rust VM
+
+This is part of a series of tutorials for building a Virtual Machine (VM):
+
+- [Introduction to VMs](./introduction-to-vm.md)
+- [How to Build a Simple Golang VM](./create-a-vm-timestampvm.md)
+- [How to Build a Complex Golang VM](./create-a-vm-blobvm.md)
+- How to Build a Simple Rust VM (this article)
+
+## Introduction
+
+The Avalanche Rust SDK is a developer toolkit composed of powerful building blocks and
+primitive types. This tutorial will walk you through the creation of a simple VM
+known as the [TimestampVM](https://github.com/ava-labs/timestampvm-rs) using the
```

Comment:
Should this TimestampVM be called `TimestampVM RS` to different from https://github.com/ava-labs/timestampvm/tree/main which is referenced in https://docs.avax.network/subnets/create-a-vm-timestampvm?